k8s.gcr.io/images/k8s-staging-secrets-store-csi-driver/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
You'll need the equivalent file in manifests, which defines the paths for promoter to operate on.

Reply:
thank you! updated it